src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1066,14 +1068,11 @@ def __getattribute__(self, name):
             if name in super().__getattribute__("_child_classes"):
                 attr._check_stable()
             return attr
-        except AttributeError as ex:
+        except AttributeError:
             self._get_parent_of_active_child_names(name)
```

Comment:
Is this still required? Please cleanup the relevant code.

Reply:
We will remove `self._get_parent_of_active_child_names(name)`.

Reply:
Done. Thank you.